src/cljs/org/broadinstitute/firecloud_ui/page/status.cljs

```diff
@@ -34,8 +34,12 @@
            [:div {} "Response text:" [:br] (-> (:xhr response) .-responseText)]]))])
    :component-did-mount
    (fn [{:keys [props state]}]
-     (utils/ajax-orch (:path (:request props))
```

Comment:
I was trying something similar that wasn't working ... will give that version a shot